packages/mobile/src/screens/notifications-screen/List.tsx

```diff
@@ -92,7 +90,7 @@ export const List = () => {
         </View>
       )}
       ListFooterComponent={() =>
```

Comment:
just fyi technically doesnt need to be anon function, just the jsx is sufficient